### ISSUE TITLE
Code Table Request - other identifier of 'volume'

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#4227]
**Goal**
Cataloged field notebook records in MVZ Archives are organized by volume IDs, and thus needs a volume identifier to house the assigned Volume. These volume identifiers are used in citations and references to the field notebooks.

**Table**
ctcoll_other_id_type

**Proposed Value**
volume ID

**Proposed Definition**
Primary organization level of a book (especially in a series), field notebook, folio, map, and other archival or library materials. Volumes may be comprised of sections or chapters. The value of this identifier is unique with respect to its collection.

**Collection type**
EH, Art

**Available for Public View**
yes

**Priority**
needed for 2022 spring semester funded project

**Comment by Jegelewicz** [comment]
This is going to be confusing - volume of a set of publications or volume (geometric) of an object? Or maybe not - it just immediately spoke to me in that way.

**Comment by mkoo** [comment]
good point-- I'll edit so our new value is "volume ID" (which was our first choice actually!)

**Comment by Jegelewicz** [comment]
I like that! and yeah, I was probably going overboard especially  since this will be for EH and Art collections. Are you guys going to create a new collection code for archives or just use EH? (off topic, just curious)

**Comment by mkoo** [comment]
We are going to send a request to Lam for ARCH
similar but not exactly the same as the others, hopefully may encourage
other museums to think about their archives too!

**Comment by Jegelewicz** [comment]
ARCH will be part of your GUID Prefix (MVZ:ARCH), right? I'm asking about the collection type - https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollection_cde

**Comment by mkoo** [comment]
EH is closest so I dont see an initial need for a new type of collection -
thanks for asking

**Comment by mkoo** [comment]
Have the CT gurus made a decision or reviewed? Can we go ahead with volume ID as other identifier? I can make the Arctos adjustments just waiting on an ok!

**Comment by Jegelewicz** [comment]
We don't meet until next week, but I don't see a problem with this.

**Comment by mkoo** [comment]
I was hasty! no worries- I will check in if I dont hear back next week!

**Comment by ccicero** [comment]
@mkoo 
This was discussed at the CT meeting 20 Jan 2022. There was concern that 'volume' is too vague and agreement that 'volume ID' is better. However, the suggestion was made to use "book volume ID" to make it clear that it's referring to some kind of book - field notebook, library book, etc.

**Comment by mkoo** [comment]
ugh-- so long winded. volume ID also covers things not strictly a book,
like folios and born-digital material
"book volume id" seems overly restrictive and wordy.... we want a code
table not a definition in every term!

**Comment by mkoo** [comment]
someone is going to ask for an example...
https://pds-imaging.jpl.nasa.gov/volumes/carto.html

**Comment by ccicero** [comment]
Hold off on final decision until Issues Meeting 3 Feb 2022.

**Comment by lin-fred** [comment]
I do think it needs to start with the word volume, no matter what clarifying values are added or not

**Comment by mkoo** [comment]
Also, remember this is for EH and Art collections so not necessarily easily confused with say, skull volumes

**Comment by Jegelewicz** [comment]
But if it is just "volume" at some point that may be useful to other collections for say, skull volume, "volume ID" would be a short but different thing from "volume".

**Comment by mkoo** [comment]
in case folks havent looked, check out the updated definition for 'volume id' in the initiating comment

**Comment by ccicero** [comment]
Looking through this thread again. So are we ok with 'volume ID' for this? I think the definition is clearer now.

**Comment by Jegelewicz** [comment]
I'm good with it.

**Comment by ccicero** [comment]
@ebraker @lin-fred  - ok to proceed with just 'volume ID' ?

**Comment by lin-fred** [comment]
I approve

**Comment by mkoo** [comment]
Thanks all. I will add the new value and definition, and close when done. We can always open a new issue if something doesnt work!
